controllers/datafoundation.go

```diff
@@ -114,3 +120,21 @@ func (r *dataFoundationReconciler) reconcileOnboardingValidationSecret() error {
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileCephIngressNetworkPolicy() error {
+	cephIngressNetworkPolicy := &netv1.NetworkPolicy{}
+	cephIngressNetworkPolicy.Name = "ceph-ingress-rule"
+	cephIngressNetworkPolicy.Namespace = r.managedFusionOffering.Namespace
+	_, err := ctrl.CreateOrUpdate(r.Ctx, r.Client, cephIngressNetworkPolicy, func() error {
+		if err := r.own(cephIngressNetworkPolicy); err != nil {
+			return err
+		}
+		desired := templates.CephNetworkPolicyTemplate.DeepCopy()
```

Comment:
I don't see a template as part of this PR. 
But the templates package is the general template package and not the datafoundation one which is a mistake.

Reply:
Corrected.